backbone.layoutmanager.js

```diff
@@ -262,11 +262,6 @@ var LayoutManager = Backbone.View.extend({
     return this.__manager__.renderDeferred.promise();
   },
 
-  // Proxy `then` for easier invocation.
-  then: function() {
-    return this.promise().then.apply(this, arguments);
```

Comment:
Ah, in that case wouldn't it better to just change our Promise resolution to not resolve to the View?